### PR TITLE
foreman_openscap 0.5.0

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 1]
```diff
@@ -23,12 +23,10 @@ License: GPLv3
 URL: https://github.com/OpenSCAP/foreman_openscap
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires: foreman >= 1.8.0
+Requires: foreman >= 1.11.0
 
 Requires: scap-security-guide
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
```
Comment (shlomizadok) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Do I need to add `Obsoletes:  %{?scl_prefix}rubygem(scaptimony)`? As we do not need it for openscap anymore

Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: You could do, but if you wish to remove the package then you should also actually delete the dir, remove it from comps/ and rel-eng/.

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 2]
```diff
@@ -15,7 +15,7 @@
 %global foreman_bundlerd_dir %{foreman_dir}/bundler.d
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.3
+Version: 0.5.0
 Release: 2%{?foremandist}%{?dist}
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: reset to 1

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec (hunk 1 above)
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: (Please do this in a separate commit.)

Comment (shlomizadok) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Will Do. One question: When I `Obsoletes: %{?scl_prefix}rubygem(scaptimony)` I don't need to specify version number(s), right?

Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: It's preferable to specify a version number in case it gets reintroduced later.

Comment (shlomizadok) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Added. Thanks

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec [hunk 3]
```diff
@@ -15,20 +15,19 @@
...
 Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 
-Requires: foreman >= 1.8.0
+Requires: foreman >= 1.11.0
 
 Requires: scap-security-guide
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
-Requires: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
-Requires: %{?scl_prefix}rubygem(scaptimony) < 0.4.0
+Obsoletes: %{?scl_prefix}rubygem(scaptimony) >= 0.3.2
```
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: It'd be `< 0.3.2-3` in this case, since you're aiming to obsolete all previously shipped versions, not future/newer versions.

(https://fedoraproject.org/wiki/Upgrade_paths_%E2%80%94_renaming_or_splitting_packages)

File: rubygem-foreman_openscap/rubygem-foreman_openscap.spec (hunk 1 above)
Comment (domcleal) [inline rubygem-foreman_openscap/rubygem-foreman_openscap.spec]: Could you also remove the package files (https://github.com/theforeman/foreman-packaging/pull/872/files#r43603837) if it's no longer needed?

**Comment by shlomizadok** [comment]
Removed Scaptimony package files.

**Comment by domcleal** [comment]
Thanks @shlomizadok, building now.
